### ISSUE TITLE
quake 1 compatibility?

### ISSUE THREAD

**Issue by lavenderdotpet** [antopilo/Nuake#10]
is full quake 1 compatibility planned

**Comment by antopilo** [comment]
Yes, a milestone for this engine would be to play quake inside of it :)

**Comment by lavenderdotpet** [comment]
hell yea! we might be able to use it as the default engine for librequake so its fully permissive licensed

**Comment by antopilo** [comment]
I was using libre quake assets internally for my test scenes already! That would be awesome. Though I have made a roadmap and I have a few main goals I still need to work on. 

Mainly I need to finish my Jolt implementation and my audio engine.

After that its skeletal animations and need a polishing pass.

**Comment by kia100** [comment]
Nuake uses a modern physics engine, which is probably different from Quake's way of abusing the hell out of real-world physics, so won't this break maps designed for the original engine?
